rakelib/changelog.rake

```diff
@@ -37,13 +37,14 @@ namespace :changelog do
 
   desc 'Check if links to issues and PRs use matching numbers between text & link'
   task :check do |task|
+    current_repo = File.basename(`git rev-parse --show-toplevel`.chomp).freeze
```

Comment:
This checks on the local file system which is more subject to change and is not really linked to GitHub.
Maybe we should use the remote instead?
Like ```File.basename(`git remote get-url origin`.chomp, '.*')```.
The tradeoff here is that it uses the `origin` remote but this is less subject to change and this is based on the real remote URL.